src/Symfony/Component/String/Tests/AbstractUnicodeTestCase.php

```diff
@@ -580,4 +580,23 @@ public static function provideReverse()
             ]
         );
     }
+
+    public static function provideWcswidth()
+    {
+        return array_merge(
+            parent::provideWcswidth(),
+            [
+                [2, '⭐'],
```

Comment:
Test data was taken from the Python implementation https://github.com/jquast/wcwidth/blob/master/wcwidth/tests/test_core.py.